tests/pages/definePassengerType.test.tsx

```diff
@@ -114,7 +129,7 @@ describe('pages', () => {
             [
                 'group',
                 {
-                    cookies: { passengerType: null },
+                    cookies: { passengerType: 'child' },
```

Comment:
The passengerType here would never be 'child' if we have the GROUP_PASSENGER_TYPES_ATTRIBUTE set. We would need this to be 'group'?